### ISSUE TITLE
Question in blogpost

### ISSUE THREAD

**Issue by MonsKim999** [anisha2102/docvqa#7]
Hi, thank you for opening great code.
I have a question from your blog post.

As you mentioned blogpost, before you fine-tune the language model (layoutlm) by docvqa dataset, you pre-trained the layoutlm using the squad dataset.

In the training phase, how to set up the 2d positional encoding? as far as I know, there are no 2d positional information in the squad dataset.

Thank you :)

**Comment by anisha2102** [comment]
Hi @MonsKim999 

I set the 2D Position Embeddings as 0 when using the SQUAD dataset. The idea behind this is - I want the model to learn robustly from both types of information (text and 2D Position) based on whatever is available. Think of it as a type of augmentation or dropout. It  also gives us the benefit of using more available datasets without being restricted by 2D Position embeddings.

Hope this helps!

**Comment by MonsKim999** [comment]
@anisha2102 
Hi, thank you for the kind explanation.
Could you let me know how much improvement when using squad pre-trained?
When you pre-train the model by squad dataset, what performance did you get from the model? Is it similar to bert-base? Or lower than bert?

In my case, it is half of the bert-base model on squad dataset. Thank you :)

**Comment by shaoqb** [comment]
@MonsKim999 
Have you reproduced the results of this project in docvqa?(0.6893)

**Comment by anisha2102** [comment]
@MonsKim999 Small correction - I did not pre-train the model on the squad dataset - I used a subset of SQUAD dataset along with the docvqa dataset while training the LayoutLM.
I observed a 2-3% increase in performance by using both datasets.
Urge you to test this and verify as it has been a few months that I worked on this project.

**Comment by MonsKim999** [comment]
@anisha2102 Thank you so much. it was very helpful to me :) 
I have trained my model using squad dataset first, then trained the model by docvqa.
Thanks a lot. have a good day !

**Comment by MonsKim999** [comment]
@shaoqb
No. I referred to the preprocessor and the learning curriculum in this repository.
